sector-builder-ffi/src/api.rs

```diff
@@ -14,6 +14,12 @@ use crate::responses::{
     FFISealedSectorHealth,
 };
 
+#[repr(C)]
+pub struct FFISealTicket {
+    height: u64,
```

Comment:
lets make these public, and add a comment what exactly height refers to

Reply:
done